src/main/java/Lesson3/Task2/Vehicle.java

```diff
@@ -0,0 +1,10 @@
+package Lesson3.Task2;
+
+public class Vehicle {
+    int height;
+    int passPlaces;
+    String port;
```

Comment:
What are passPlaces and port parameters?

Reply:
5 parameters should be here:
![image](https://user-images.githubusercontent.com/36937046/216822898-079d8f24-6dec-4e28-839b-10d4b71a5515.png)